### PR TITLE
Create Palindromer

### PR DESCRIPTION
**Pull request by bsztukowski**
Copied code I worked on this morning from visual studio. Check it out, Derek!

### CODE REVIEW

File: Palindromer [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+        
+        static List<string> FindPalandromes(string s, bool ignore)
+        {
+            //declare a list to populate with all palandromes from the input string
+            List<string> palandromeList = new List<string>(); 
+
+            //Alter string to ignore spaces when digging for palandromes, if user so chooses
+            if (ignore == true)     
+            {
+                s = s.Replace(" ", "");
```
Comment (derklee91) [inline Palindromer:35]: Instead of "" try String.Empty

File: Palindromer [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Palandromer
+{
+    class Program
+    {
+        //Function to determine whether a string is a palandrome (uses the FindPalandromes method, since I wrote it first)
+        static bool IsPalandrome(string s, bool ignore)
```
Comment (derklee91) [inline Palindromer:13]: I think we should make a new class called PalandromeUtils and move the palandrome methods into that class.

File: Palindromer [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+                    }
+                }
+            }
+            return palandromeList;
+            
+        }
+        
+        static void Main(string[] args)
+        {
+
```
Comment (derklee91) [inline Palindromer:81]: I would remove any open spaces after an open curly bracket {

**Review by derklee91 — APPROVED** [review]

File: Palindromer (hunk 1 above)
Comment (bsztukowski) [reply Palindromer:35]: Ok; trying to make a branch to edit and do this right 🗡️ haha

**Pull request merged**